packages/core-database/lib/wallet-manager.js

```diff
@@ -183,20 +184,63 @@ module.exports = class WalletManager {
     Object.values(this.byAddress).map(wallet => (wallet.dirty = false))
   }
 
+  /**
+   * Load a list of all active delegates.
+   * @param  {Number} maxDelegates
+   * @return {Array}
+   */
+  loadActiveDelegateList (maxDelegates, height) {
+    if (height > 1 && height % maxDelegates !== 1) {
+      throw new Error('Trying to build delegates outside of round change')
+    }
+
+    const { round } = roundCalculator.calculateRound(height, maxDelegates)
+    let delegates = this.allByUsername()
+
+    if (delegates.length < maxDelegates) {
+      throw new Error(`Expected to find ${maxDelegates} delegates but only found ${delegates.length}. This indicates an issue with the genesis block & delegates.`)
+    }
+
+    delegates = delegates.sort((a, b) => {
+      const aBalance = +a.voteBalance.toFixed()
```

Comment:
by the way what is the point of this? i guess keeping a.voteBalance is enough.
- they are numbers
- if there is a rounding issue, since they are all integers, it won't change the order

Reply:
`a.voteBalance` is a bignumber object inside a wallet object.

Reply:
We could use https://mikemcl.github.io/bignumber.js/#cmp

Reply:
ah true that, i should keep that in mind now. Is there some kind of comparison stuff in bignumber library, sounds weird to cast down to normal number in order to compare again

Reply:
http://mikemcl.github.io/bignumber.js/#cmp